src/Pokemon.java

```diff
@@ -0,0 +1,50 @@
+import java.util.List;
+
+public abstract class Pokemon {
+    String name;
+    Integer level;
+    Integer hp;
+    String food;
+    String sound;
+    String type;
```

Comment:
Je geeft dit type nooit een waarde. In de subklassen implementeer je wel de getter, maar die laat je gewoon de String "water" of "grass" returnen. Beter is om deze waarde echt in te vullen in de constructor van de subklasse. Dat kan ook met `super.setType("water")` of door het aan de parameters van de super toe te voegen.